omnibus-arbitrum-sepolia.toml

```diff
@@ -0,0 +1,186 @@
+name = "synthetix-omnibus"
+version = "1"
+description = "Arbitrum deployment"
+preset = "main"
+include = [
+    "tomls/omnibus-arbitrum-sepolia/permissions.toml",
+
+    "tomls/core.toml",
+    "tomls/settings.toml",
+    "tomls/permissions.toml",
+    "tomls/permit-deniers.toml",
+    "tomls/pools/spartan-council.toml",
+    "tomls/markets/spot-factory.toml",
+    "tomls/markets/perps-factory.toml",
+    "tomls/markets/common/bigcap-settings.toml",
+
+    # USDC
+    "tomls/oracles/pyth-usdc.toml",
+    "tomls/omnibus-arbitrum-sepolia/spot/usdc.toml",
+    "tomls/omnibus-arbitrum-sepolia/collaterals/usdc.toml",
+
+    # DAI
+    "tomls/oracles/pyth-dai.toml",
+    "tomls/omnibus-arbitrum-sepolia/spot/dai.toml",
+    "tomls/omnibus-arbitrum-sepolia/collaterals/dai.toml",
+]
+
+[setting.commitmentPriceDelay]
+defaultValue = "2"
+
+[setting.bigCapSettlementDelay]
+defaultValue = "2"
+
+[setting.snx_package]
+defaultValue = "synthetix:3.3.15"
+
+[setting.perps_keeper_cost_package]
+defaultValue = "arbitrum-gas-price-oracle:3.3.16"
+
+[setting.spot_market_package]
+defaultValue = "synthetix-spot-market:3.3.15"
+
+[setting.perps_market_package]
+defaultValue = "synthetix-perps-market:3.3.19"
+
+[setting.buyback_snx_package]
+defaultValue = "buyback-snx:3.3.15"
+description = "Do not update! This package isn't upgradeable. A version change will deploy a new contract."
+
+[setting.pyth_price_verification_address]
+defaultValue = "0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF"
+description = "https://docs.pyth.network/documentation/pythnet-price-feeds/evm"
+
+[setting.pythFeedUrl]
+defaultValue = "https://api.synthetix.io/pyth-mainnet/api/get_vaa_ccip?data={data}"
+description = "Pyth settlement strategy config"
+
+[provision.pyth_erc7412_wrapper]
+source = "pyth-erc7412-wrapper:3.3.15"
+targetPreset = "<%= settings.target_preset %>"
+options.salt = "<%= settings.salt %>"
+options.pythAddress = "<%= settings.pyth_price_verification_address %>"
+
+[setting.owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.deployer]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.minimum_liquidity_ratio]
+defaultValue = "<%= parseEther('2.00') %>"
+description = "200% min market collateralization"
+
+[setting.account_timeout_withdraw]
+defaultValue = "<%= String(1 * 24 * 3600) %>"
+
+[setting.sc_pool_id]
+defaultValue = "1"
+
+[setting.pool_owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.settlementReward]
+defaultValue = "1"
+
+[setting.settlementMinimumUsdExchangeAmount]
+defaultValue = "<%= parseEther('0.000001') %>"
+
+[setting.settlementMaxRoundingLoss]
+defaultValue = "<%= parseEther('0.000001') %>"
+
+[setting.priceDeviationTolerance]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.spotStrictStalenessTolerance]
+defaultValue = "60"
```

Comment:
This can probably be looser (or bigCapDefaultStalenessTolerance, whatever value is being used by the spot market to determine the value)

Reply:
The setting is strict and big cap strict is 60. 
this is what was set for snx spot on OP. 
why would we switch to default (3600) here?

Reply:
DAI/USDC is expected to fluctuate significantly less than other assets, so just making sure we have a price that's as recent as like a day (maybe even a week?) should be okay? Like if there's a big depeg, presumably someone will be incentivized to post a more recent price to Pyth. It's a risk parameter though, not completely sure the best way to pick a setting, just intuitively seems like it should be looser than anything we've used so far.

Reply:
Makes sense. Intuitively. But *practically* what this value should be?

Reply:
settled to use separate tolerance settings and both set to 3600